<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="1" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="13444" started-at="2018-12-27T02:31:43Z" finished-at="2018-12-27T02:31:56Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="13444" started-at="2018-12-27T02:31:43Z" finished-at="2018-12-27T02:31:56Z">
      <class name="day7.GuruProject">
        <test-method status="PASS" signature="invokeBrowser()[pri:0, instance:day7.GuruProject@6cc4c815]" name="invokeBrowser" is-config="true" duration-ms="11688" started-at="2018-12-27T02:31:43Z" finished-at="2018-12-27T02:31:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invokeBrowser -->
        <test-method status="FAIL" signature="verifyTitleOfThePage()[pri:0, instance:day7.GuruProject@6cc4c815]" name="verifyTitleOfThePage" duration-ms="208" started-at="2018-12-27T02:31:55Z" finished-at="2018-12-27T02:31:55Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Guru99 Bank Home Page1] but found [Guru99 Bank Home Page]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Guru99 Bank Home Page1] but found [Guru99 Bank Home Page]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at day7.GuruProject.verifyTitleOfThePage(GuruProject.java:27)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTitleOfThePage -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:day7.GuruProject@6cc4c815]" name="closeBrowser" is-config="true" duration-ms="1510" started-at="2018-12-27T02:31:55Z" finished-at="2018-12-27T02:31:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- day7.GuruProject -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
